src/smart-components/access-management/UsersTable.tsx

```diff
@@ -72,18 +79,31 @@ const UsersTable: React.FunctionComponent = () => {
     }
   };
 
-  const rows = users.map((user: UserProps) => [
-    user.username,
-    user.email,
-    user.first_name,
-    user.last_name,
-    user.is_active ? 'Active' : 'Inactive',
-    user.is_org_admin ? 'Yes' : 'No',
-    {
-      cell: <ActionsColumn items={ROW_ACTIONS} />,
-      props: { isActionCell: true },
-    },
-  ]);
+  const rows = useMemo(() => {
+    return users.map((user: any) => [
+      user.username,
+      user.email,
+      user.first_name,
+      user.last_name,
+      user.is_active ? 'Active' : 'Inactive',
+      user.is_org_admin ? 'Yes' : 'No',
+      {
+        cell: (
+          <ActionsColumn
+            items={[
+              { title: 'Add to user group', onClick: () => console.log('ADD TO USER GROUP') },
```

Comment:
Can you please make the strings localized?

Reply:
fixed